politeiawww/www.go

```diff
@@ -464,7 +470,14 @@ func (p *politeiawww) handleSetProposalStatus(w http.ResponseWriter, r *http.Req
 	}
 	defer r.Body.Close()
 
-	reply, err := p.backend.ProcessSetProposalStatus(sps)
+	user, err := p.getSessionUser(r)
+	if err != nil {
+		RespondWithError(w, r, 0,
+			"handleNewComment: getSessionUser %v", err)
```

Comment:
It's like you're trying to make it harder to read the error logs. This should be handleSetProposalStatus, not handleNewComment.